aries-site/src/pages/templates/wizard.mdx

```diff
@@ -0,0 +1,66 @@
+import { Example } from '../../layouts';
+import { SimpleWizardExample, TwoColumnWizardExample } from '../../examples';
+
+<Example
+  code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/templates/wizard/SimpleWizardExample.js"
+  figma="https://www.figma.com/file/GEhgYzx3b8MejBjg9gAxyA/Wizarding?node-id=1%3A2"
+  screenContainer
+  width="100%"
+>
+  <SimpleWizardExample />
+</Example>
+
+## Guidance
+
+Use a wizard when your form involves multiple steps. This makes the experience more manageable for the user.
```

Comment:
while helping to limit user error ?

Reply:
Adding in that a good way to deal with long forms is to break them down into multiple steps

Reply:
Good idea. I will add something here. I think we also need to add some guidance around validation and how validation occurs in a wizard.

Reply:
Yeah I was going to ask about validation as well

Reply:
I've added some initial guidance on validation but still need to add it into the examples themselves.